### PR TITLE
Chat app

### PR DESCRIPTION
**Pull request by toddpla**

### CODE REVIEW

File: app/assets/javascripts/cable/subscriptions/chat.coffee [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+App.chatChannel = App.cable.subscriptions.create { channel: "ChatChannel", room: "Group Chat"},
+  received: (data) ->
+    @appendLine(data)
+
+  appendLine: (data) ->
+    html = @createLine(data)
+    # $("[data-chat-room='Group Chat']").append(html)
```
Comment (runticle) [inline app/assets/javascripts/cable/subscriptions/chat.coffee]: remove da comment

File: app/assets/javascripts/cable/subscriptions/chat.coffee [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    """
+    <article>
+      <div class="chat-line border border-primary rounded">
+        <span class="speaker">#{data["sent_by"]}</span><br>
+        <span class="body">#{data["body"]}</span>
+      </div>
+    </article>
+    """
+
+App.chatChannel.send({ sent_by: "Paul", body: "This is a cool chat app." })
```
Comment (imogenmisso) [inline app/assets/javascripts/cable/subscriptions/chat.coffee]: It is a very cool chat app.

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@
 module SessionsHelper
   def log_in(user)
     session[:user_id] = user.id
+    cookies[:user_id] = user.id
```
Comment (imogenmisso) [inline app/helpers/sessions_helper.rb]: 🌟

File: spec/models/chat_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Chat, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (imogenmisso) [inline spec/models/chat_spec.rb]: can delete this

File: spec/helpers/chats_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+#
+# describe ChatsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ChatsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (imogenmisso) [inline spec/helpers/chats_helper_spec.rb]: can delete

File: app/assets/javascripts/cable/subscriptions/chat.coffee (hunk 2 above)
Comment (shannongamby) [reply app/assets/javascripts/cable/subscriptions/chat.coffee]: ⭐️

**Comment by runticle** [comment]
⭐️

**Pull request merged**